.github/workflows/scripts/check_clang_tidy.sh

```diff
@@ -9,4 +9,4 @@ rm -rf build && mkdir build && cd build
 cmake $CI_SETUP_CMAKE_ARGS ..
 
 cd ..
-python3 ./scripts/run_clang_tidy.py $PWD/taichi -clang-tidy-binary clang-tidy-10 -checks=-*,performance-inefficient-string-concatenation -header-filter=$PWD/taichi -p $PWD/build -j2
+python3 ./scripts/run_clang_tidy.py $PWD/taichi -clang-tidy-binary clang-tidy-10 -checks=-*,performance-inefficient-string-concatenation,readability-identifier-naming -header-filter=$PWD/taichi -p $PWD/build -j2
```

Comment:
OOC: is `=$PWD/taichi` necessary after the rule?

Reply:
clang-tidy checks the directory specified here. Our cpp files are mainly in this folder, right?

Reply:
sorry I was dumb :P